components/headers/Header.style.ts

```diff
@@ -0,0 +1,21 @@
+import device from '@utils/breakpointsDevice';
+import styled from 'styled-components';
+
+export const Header = styled.div`
+  height: 70px;
+  margin: 0 auto;
+  padding: 0 80px 0 26px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: var(--white-white_FFFFFF, #fff);
```

Comment:
```suggestion
  background: var(--white);
```
마찬가지로 globals.css 확인 부탁드려요!